oarepo_ui/templates/components/datafields/ISection.jinja

```diff
@@ -0,0 +1,10 @@
+{# def title=None, title_hidden=False #}
+
+<div class="ui cf-section">
```

Comment:
would it be possible to name the component without the cf- prefix? The reason is that the section field can be used outside the custom fields as well (and is used in mbdb)